nextflow_schema.json

```diff
@@ -641,6 +641,11 @@
                     "fa_icon": "fas fa-file",
                     "description": "Index file for `bcftools_annotations`"
                 },
+                "bcftools_columns": {
+                    "type": "string",
```

Comment:
I'm not sure why `bcftools_header_lines` below is a string and not a file; I'm following that convention here